lib/glific/templates/interactive_templates.ex

```diff
@@ -113,17 +113,61 @@ defmodule Glific.Templates.InteractiveTemplates do
   @spec create_interactive_template(map()) ::
```

Comment:
In `list` type interactive messages, if i add markdown in list title / description the interactive message is not getting send. Can you check once?.
<img width="720" alt="Screenshot 2024-09-03 at 11 01 22 AM" src="https://github.com/user-attachments/assets/4282eb7a-dc5a-4adf-a8d8-fe337cca1e18">